openapi/main.yaml

```diff
@@ -0,0 +1,704 @@
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                    description: Details about the error
+                example:
+                  status: 400
+                  error: Invalid request path.
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                example:
+                  status: 500
+                  error: An error occurred while processing this request.
+              example:
+                status: 500
+                error: An error occurred while processing this request.  
+  /payments:
+    get:
+      tags:
+        - Payments
+      summary: Get paginated list of payments
+      description: Get a paginated list of incoming and outgoing payments of a stellar account
+      operationId: GetPayments
+      parameters:
+        - name: address
+          in: query
+          description: The stellar address whose payments we want to fetch.
+          required: true
+          schema:
+            type: string
+        - name: afterId
+          in: query
+          description: The starting operation id of the list of payments 
+          required: false
+          schema:
+            type: string
+        - name: beforeId
+          in: query
+          description: The ending operation id of the list of payments.
+          required: false
+          schema:
+            type: string
+        - name: sort
+          in: query
+          description: sort oder.
+          required: false
+          schema:
+            type: string
+            enum:
+              - ASC
+              - DESC
+            default: DESC
+        - name: limit
+          in: query
+          description: number of payments to return. Default is 50, maximum is 200
+          required: false
+          default: 50
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Returns a list of payments and urls to fetch further payments
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  _links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                      next:
+                        type: string
+                      prev:
+                        type: string
+                  payments:
+                    type: array
+                    description: list of payments 
+                    items:
+                      type: object
+                      properties:
```

Comment:
should it cover some [path payment properties](https://developers.stellar.org/docs/learn/fundamentals/transactions/list-of-operations#payment) like the below?
- `path` (e.g. ["XLM", "BTC", "USDC"]
- `destinationMin`
- `sendMax
- and maybe a `kind` prop (e.g. PAYMENT | PATH_PAYMENT_STRICT_SEND | PATH_PAYMENT_STRICT_RECEIVE)

Also, I am assuming we are not supporting Anchor Transaction properties on those payment objects. Is that right?
E.g.: `anchor transaction id`,  `kind`, `status`, `more_info_url`, etc...

Reply:
All good points @CassioMG. And there's certainly other things that I think need to change in this API as well. I'm going to start a doc with all these recommendations